styles.css

```diff
@@ -67,15 +86,50 @@ form input {
     font-size: 1.4rem;
     width: 28%;
     line-height: 35px;
-    border: 1px solid grey;
+    border: 1px solid #808080;
     border-radius: 3%;
     background-color: #d6d6d6;
 }
 
 .disease-item img {
     width: 100%;
     height: 280px;
-    border: 1px solid black;
+    border: 1px solid #000000;
     border-radius: 3%;
 
 }
+
+
+input[type="file"] {
+    cursor: pointer;
+    /*background-color: #decaff;*/
+    /*height: 40px;*/
+    /*font-size: 2rem;*/
+}
+
+input[type="submit"] {
+    display: block;
+    background-color: #d6d6d6;
+    height: 40px;
+    font-size: 1.5rem;
+    cursor: pointer;
+}
+
+input[type="submit"]:hover {
+    background-color: #6e6e6e;
+}
+
+
+
+
+.submit-remove {
+    margin-top: 30px;
+}
+
+
+/*.submit-remove{*/
```

Comment:
Commented code doesn't belong in pull requests - nobody will know why this has been commented and it'll just stay there "in case someone needs it" for years to come